src/geoserver/util.py

```diff
@@ -455,6 +455,40 @@ def delete_db_store(geoserver_workspace, auth):
         response.raise_for_status()
 
 
+def patch_coverage(geoserver_workspace, layer, coverage_store, *, title=None, description=None, bbox=None, auth):
+    coverage = dict()
+
+    if title:
+        coverage['title'] = title
+        keywords = [
+            "features",
+            layer,
+            title
+        ]
+        keywords = list(set(keywords))
+        coverage['keywords'] = {
+            "string": keywords
+        }
+    if description:
```

Comment:
`if description` vs. `if description is not None` - not sure which is more suitable